WebRoot/templates/calendar/Appointment.template

```diff
@@ -239,6 +239,7 @@
 
 <!-- NOTE: Hopefully the EditView template can go away as soon as task -->
 <!-- NOTE: compose is redesigned to be more like appt compose. -->
+<!-- TODO: Edit View template for Appointment and Task are now separated. Remove conditional code for task-edit.-->
```

Comment:
I guess we already have ticket in pipeline for this. When we start working on appointment create form, we will have to visit this code.